intents/slice-compare.py

```diff
@@ -0,0 +1,66 @@
+"""This module currently contains the signature/implementation of some intents"""
+
+from Utilities import aspects
+
+def slice_compare(table, metric, dimension, slices, summary_operator, **kwargs):
+    """This function will implement the slice-compare intent
+
+    Sorts tuples in the order metric, after applying slice, groupby operations.
+    Also removes the tuples that do not lie in the given date range.
+    The arguments 'table, metric,dimension, slices, summary_operator' are not optional so they are passed as it is,
```

Comment:
Limit the length of the line to 80 chars, similarly everywhere. http://google.github.io/styleguide/pyguide.html#32-line-length